### ISSUE TITLE
No stream since firmware update

### ISSUE THREAD

**Issue by napo-leon** [senorshaun/homebridge-lollipop#1]
Hello I Just would like to know if someone are able to use this plugin since last firmware update of the camera? Or the issue is on my side! 

Thanks a lot for your work!!

**Comment by senorshaun** [comment]
Definitely not on your side. Withe started requiring authentication for the Mqtt connection and rtsp stream. I'm working on finding out what passwords they are using but its taking me a while

**Comment by TineTurk** [comment]
Any updates here?

**Comment by senorshaun** [comment]
I figured out how to authenticate the rtsp stream. But the Mqtt connection uses a cert based hand shake I haven’t figured out. If someone else gets that figured out, I can incorporate it and get this working again

**Comment by asiangavin** [comment]
After I found the direct RTSP stream using linux and MQTT ([using some pointers in this link](https://www.lafois.com/2022/02/04/lollipop-camera-reverse-engineering/)) I can direct open the camera feed in VLC and also the using the HomeKit plugin [FFmpeg camera plugin](https://github.com/Sunoo/homebridge-camera-ffmpeg#readme).  I can get it to work in FFmpeg but not your Lollipop plugin. Maybe related?

**Comment by senorshaun** [comment]
Yeah it is. I got the rtsp stream authentication reverse engineered, so to dynamically get the video stream all I need is the camera id. Previously I was getting that through mqtt. However mqtt is now using auth, a server and client cert handshake with a key that I haven’t figured out because I don’t have a way to sniff it properly and decompiling the android apk only got me so far. 
I could still get the client I’d if I did a login to api.lollipop.camera using credentials people enter, then pull cameras and camera ids and local ip addresses. It wouldn’t be too hard to do that. 

What settings do you have ffmpeg configured with? I never got audio working smoothly before the firmware update

**Comment by asiangavin** [comment]
I don't think I have done anything except default settings for ffmpeg camera. Audio is a little patchy cutting in and out like you said but it does work.
Video source: -i rtsp://x.x.x.x:554/live/<camera id>
Enable audio checked
Unbridge camera checked

**Comment by burritobill** [comment]
Any update on this?

> I could still get the client I’d if I did a login to api.lollipop.camera using credentials people enter, then pull cameras and camera ids and local ip addresses. It wouldn’t be too hard to do that.

This sounds like a reasonable solution.

**Comment by mitch7391** [comment]
@senorshaun did you have any luck moving forward with this?

**Comment by seanmckeever91** [comment]
@senorshaun any luck? Would love to get this camera into homebridge

**Comment by jakedeg** [comment]
Just bumping this again @senorshaun
